biothings/web/analytics/channels.py

```diff
@@ -1,83 +1,99 @@
+import aiohttp
+import asyncio
 import certifi
+import logging
 import orjson
-from tornado.httpclient import HTTPRequest
 
 from biothings.web.analytics.events import Event, Message
 
-
 class Channel:
-    def handles(self, event):
+    async def handles(self, event):
         raise NotImplementedError()
 
-    def send(self, event):
+    async def send(self, event):
         raise NotImplementedError()
 
 
 class SlackChannel(Channel):
     def __init__(self, hook_urls):
         self.hooks = hook_urls
 
-    def handles(self, event):
+    async def handles(self, event):
         return isinstance(event, Message)
 
-    def send(self, message):
-        for url in self.hooks:
-            yield HTTPRequest(
-                url=url,
-                method="POST",
-                headers={"content-type": "application/json"},
-                body=orjson.dumps(message.to_slack_payload()).decode(),
-                ca_certs=certifi.where(),  # for Windows compatibility
-            )
-
+    async def send(self, event):
+        async with aiohttp.ClientSession() as session:
+            tasks = [self.send_request(session, url, event) for url in self.hooks]
+            await asyncio.gather(*tasks)
 
-# Measurement Protocol (Universal Analytics)
-# https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide
+    async def send_request(self, session, url, event):
+        async with session.post(
+                url,
+                json=event.to_slack_payload(),
+                verify=certifi.where()  # for Windows compatibility
+        ) as _:
+            pass
 
 
 class GAChannel(Channel):
     def __init__(self, tracking_id, uid_version=1):
         self.tracking_id = tracking_id
         self.uid_version = uid_version
 
-    def handles(self, event):
+    async def handles(self, event):
         return isinstance(event, Event)
 
-    def send(self, payload):
+    async def send(self, payload):
```

Comment:
`arguments-renamed: Parameter 'event' has been renamed to 'payload' in overriding 'GA4Channel.send' method`. Not essential just highlighting the API change for different channels. This existed before your PR as well